ffconv/stream_processors.py

```diff
@@ -108,9 +108,15 @@ def _init_stream(self, stream, profile):
         # Set input reference frames value
         self.refs = int(stream['refs'])
 
-        # Set target values for frames, profile, level, preset and quality
+        # Get height and set target for ref frames (default is 4)
+        self.max_refs = 4
         height = int(stream.get('height', 720))
```

Comment:
Woah, are we sure about that? I actually think we should raise an Exception if we can't find the height.